dev-docs/bidders/minutemedia.md

```diff
@@ -4,7 +4,7 @@ title: MinuteMedia
 description: Prebid Bidder Adaptor
```

Comment:
If you find the time you find a more descriptive description ;)

Reply:
I've change a little bit the description. reviewed the others adapters and it's looks almost the same